web/client/plugins/Map.jsx

```diff
@@ -426,5 +426,8 @@ module.exports = {
         maptype: require('../reducers/maptype'),
         additionallayers: require('../reducers/additionallayers')
     },
-    epics: require('../epics/map')
+    epics: {
+        ...require('../epics/map'),
+        ...require('../epics/mapInfo')
```

Comment:
whops... i didn't see this. Please epics should be imported by the identify plugins, and the epic should be part of it.